src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -27,7 +27,7 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
     );
   }
 
-  if (type === 'COMMUNITY') {
+  if (type === 'TOPIC') {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from checking if `type === 'COMMUNITY'` to `type === 'TOPIC'` might cause issues if there are still components that pass the type as 'COMMUNITY'. Please ensure that all instances where this component is used have been updated accordingly.
```diff
-  if (type === 'COMMUNITY') {
+  if (type === 'TOPIC') {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->